# thumb_prox_joint_effort
# thumb_dist_joint_effort
# finger1_roll_joint_position
# finger1_prox_joint_effort
# finger1_dist_joint_effort
# finger2_roll_joint_position
# finger2_prox_joint_effort
# finger2_dist_joint_effort

float64[] commands  # array of the above commands
